README.md

```diff
@@ -34,61 +27,69 @@ cd langstar
 cargo install --path cli
 ```
 
-### Configuration
+### Configuration Quick Start
 
-Set your API key via environment variable:
+> **⚠️ Important**: Different Langstar commands require different API keys and configuration.
 
-```bash
-export LANGSMITH_API_KEY="your-api-key-here"
-```
+Langstar interacts with **two distinct LangChain services**, each with different configuration requirements:
 
-Or create a config file at `~/.config/langstar/config.toml`:
+#### For LangSmith Prompts (`langstar prompt *`)
 
-```toml
-langsmith_api_key = "your-api-key-here"
-output_format = "table"
-```
+**Required:**
+- `LANGSMITH_API_KEY` - Your LangSmith API key ([get one here](https://smith.langchain.com))
+
+**Optional (for organization/workspace scoping):**
+- `LANGSMITH_ORGANIZATION_ID` - Scope operations to a specific organization
+- `LANGSMITH_WORKSPACE_ID` - Scope operations to a specific workspace
 
-#### Organization and Workspace Scoping
+**Example:**
+```bash
+# Minimal setup (personal prompts)
+export LANGSMITH_API_KEY="<your-api-key>"
+langstar prompt list
 
-Langstar supports scoping operations to a specific organization or workspace. This is useful when working with team prompts or enterprise deployments.
+# With workspace scoping (team prompts)
+export LANGSMITH_API_KEY="<your-api-key>"
+export LANGSMITH_WORKSPACE_ID="<your-workspace-id>"
+langstar prompt list
+```
 
-**Configuration Methods:**
+#### For LangGraph Assistants (`langstar assistant *`)
 
-1. **Environment Variables:**
-   ```bash
-   export LANGSMITH_ORGANIZATION_ID="your-org-id"
-   export LANGSMITH_WORKSPACE_ID="your-workspace-id"
-   ```
+**Required:**
+- `LANGGRAPH_API_KEY` - Your LangGraph Cloud API key (or `LANGSMITH_API_KEY` as fallback)
```

Comment:
`LANGGRAPH_API_KEY` is the same as `LANGSMITH_API_KEY`. We should drop support for `LANGGRAPH_API_KEY` and require `LANGSMITH_API_KEY`.